metals/src/main/scala/scala/meta/internal/metals/debug/ExtendedScalaMainClass.scala

```diff
@@ -61,10 +70,68 @@ object ExtendedScalaMainClass {
     s"$safeJavaHome $jvmOptsString -classpath \"$classpathString\" $mainClass $argumentsString"
   }
 
+  /**
+   * Create a manifest jar containing the classpath.
+   * It's created in `.metals/tmp` woth name derived
```

Comment:
Shouldn't it be created in proper `tmp` directory which is being cleaned from time to time? Is `.metals/tmp` cleaned or will it grow and grow?

Reply:
It should delete on exit and it seems to work when testing locally.